example/app/controllers/authenticated_controller.rb

```diff
@@ -1,5 +0,0 @@
-class AuthenticatedController < ApplicationController
```

Comment:
are we not providing any of this stuff for them anymore, or did it just get moved elsewhere?